Revit_Engine/Create/Settings/MappingSettings.cs

```diff
@@ -35,46 +34,48 @@ public static partial class Create
         /***************************************************/
         /****              Public methods               ****/
         /***************************************************/
-
+        
+        [PreviousVersion("4.3", "BH.Engine.Adapters.Revit.Create.ParameterSettings(System.Collections.Generic.IEnumerable<BH.oM.Adapters.Revit.Mapping.ParameterMap>, System.Collections.Generic.IEnumerable<BH.oM.Adapters.Revit.Mapping.FamilyMap>, System.String, System.String)")]
         [Description("Creates an entity holding information about the enforced convert relationships between Revit families and BHoM types on Pull as well as mapping between Revit parameters and BHoM object properties.")]
         [InputFromProperty("parameterMaps")]
         [InputFromProperty("familyMaps")]
         [InputFromProperty("tagsParameter")]
         [InputFromProperty("materialGradeParameter")]
-        [Output("parameterSettings")]
-        public static ParameterSettings ParameterSettings(IEnumerable<ParameterMap> parameterMaps = null, IEnumerable<FamilyMap> familyMaps = null, string tagsParameter = "", string materialGradeParameter = "")
+        [Output("mappingSettings")]
+        public static MappingSettings MappingSettings(IEnumerable<ParameterMap> parameterMaps = null, IEnumerable<FamilyMap> familyMaps = null, string tagsParameter = "", string materialGradeParameter = "")
```

Comment:
This would be classed as a new method for the null reports, so if not handling nulls appropriately, will flag as an error not existing on `master`/`main`